src/api/pokemon.ts

```diff
@@ -0,0 +1,15 @@
+import {PokemonClient} from 'pokenode-ts';
+
+// getBerry returns berry info
+// TODO currently just prints stuff to console
+export const getPokemonList = async () => {
+  const api = new PokemonClient({
+    cacheOptions: {maxAge: 5000, exclude: {query: false}},
+  });
+  await api
+    .listPokemons(0, 1200)
+    .then(data => {
+      return data.results;
+    })
+    .catch(error => console.error(error));
```

Comment:
so currently this doesn't actually return the results, you would need to do something like

```suggestion
  return await api
    .listPokemons(0, 1200)
    .then(data => {
      return data.results;
    })
    .catch(error => console.error(error));
```

or:

```suggestion
  const result = await api
    .listPokemons(0, 1200)
    .then(data => {
      return data.results;
    })
    .catch(error => console.error(error));
  // do stuff with result
  return result
```